src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -317,18 +317,21 @@ export const VaultDetailPanels = ({
         </VaultActions>
       </Row>
 
-      {chartData && (
+      {chartData && chartValue && (
         <VaultChart>
           <StyledCardHeader header={t('vaultdetails:performance-panel.header')} />
 
           <ChartValueContainer>
             <ChartValueLabel>{t('vaultdetails:performance-panel.earnings-over-time')}</ChartValueLabel>
-            <ChartValue>{formatUsd(chartValue)}</ChartValue>
+            <ChartValue>
+              {formatAmount(chartValue, selectedVault?.token?.decimals)} {selectedVault?.token?.symbol}
```

Comment:
I didn't see a great way in `utils/` to format a BN to a certain number of decimal points, can do it manually inline with BN but wanted to see if there was preferred way to do it